tests/network-tests/src/scenarios/full.ts

```diff
@@ -60,6 +61,7 @@ scenario('Full', async ({ job, env }) => {
   // Council (should not interrupt proposalsJob!)
   const secondCouncilJob = job('electing second council', electCouncil).requires(coreJob)
   const councilFailuresJob = job('council election failures', failToElect).requires(secondCouncilJob)
+  job('council election failure with blacklist', failToElectWithBlacklist).requires(secondCouncilJob)
```

Comment:
Will there be two council jobs running unless..
```suggestion
  job('council election failure with blacklist', failToElectWithBlacklist).requires(councilFailuresJob)
```